e2e/helpers/date_to_string_helper.js

```diff
@@ -1,2 +1,15 @@
-module.exports = (date) => date.year + '-' + date.month + '-' + date.day + ' ' +
-  date.hour + ':' + date.minute + ':' + date.second;
+module.exports = (date) => {
+  let hour = '00', minute = '00', second = '00';
+
+  if (date.hour) {
+    hour = date.hour;
+  }
+  if (date.minute) {
+    minute = date.minute;
+  }
+  if (date.second) {
+    second = date.second;
+  }
```

Comment:
Think this section might be able to be simplified and easily expanded for the `year`, `month`, and `day`.
I think that each variable could be assigned as so:
```js
const hour = date.hour || '00';
```

Reply:
actually could be simplified even further if you have a method like
```js
const getValue = (value) => value || '00';
```
which and in the exported method you could just do
```js
(date) => getValue(date.year) + '-' + getValue(date.month) + '-' + getValue(date.day) + ' ' + getValue(date.hour) + ':' + getValue(date.minute) + ':' + getValue(date.second);
```

Reply:
Name could be improved on the new method obviously

Reply:
👍